src/base/static/components/ui-elements/warning-messages-container.js

```diff
@@ -11,10 +13,10 @@ const WarningMessagesContainer = props => {
         "warning-messages-container--visible": props.errors.length > 0,
```

Comment:
As we discussed earlier, having a `--visible` modifier is an anti-pattern. This means that we are rendering this container, and setting its display to `none` by default, which makes no sense.

Instead, I highly suggest removing this classnames logic and having the parent component render `WarningMessagesContainer` like so:

```
{ errors.length && <WarningMesssagesContainer errors={errors}/> }
```

That way we can rely on React's virtual DOM diffing, instead of CSS flags, to handle the logic of whether or not to display warning messages.

Also, why are we wrapping this component in a `section` tag? What's wrong with `div`?